minio/Dockerfile

```diff
@@ -15,8 +15,8 @@ COPY --from=builder /usr/bin/minio /usr/bin/minio
 COPY --from=builder /usr/bin/docker-entrypoint.sh /usr/bin/docker-entrypoint.sh
 
 RUN  \
-     apk add --no-cache ca-certificates 'curl>7.61.0' 'su-exec>=0.2' && \
-     echo 'hosts: files mdns4_minimal [NOTFOUND=return] dns mdns4' >> /etc/nsswitch.conf
+    apk add --no-cache ca-certificates curl 'su-exec>=0.2' && \
```

Comment:
Please also add an upgrade of the alpine packages to get latest security fixes, see as an example here: https://github.com/kyma-incubator/third-party-images/blob/master/k8s-tools/Dockerfile